.github/workflows/reusable-widget-e2e.yml

```diff
@@ -84,6 +61,11 @@ jobs:
       - name: Wait on WIDGET
         run: wait-on --timeout=90000 http://localhost:3500
 
+      - name: Cypress install
+        if: steps.setup-project.outputs.cypress_cache_hit != 'true'
+        working-directory: apps/widget
+        run: pnpm cypress install
+
```

Comment:
Were we missing it before?

Reply:
We now cache the cypress binaries so it was needed now :)

Reply:
hehe, I noticed the warning about it some time ago but totally forgot about it 😅